README.md

```diff
@@ -16,10 +16,17 @@ export NR_ACCOUNT_ID=<New Relic account id>
 export PIXIE_API_TOKEN=<Pixie api token>
 export PIXIE_CLUSTER_ID=<Pixie cluster id>
 ``` 
+
+(Optional) If you would like to set up monitoring with New Relic, save your license key to your environment:
```

Comment:
We might want to be extra clear about this since this plugin already sends data to NR regardless of this license key and I could see users getting confused. You might phrase this like:
> (Optional) If you would like to set up monitoring of the plugin with New Relic, save your license key to your environment:

Also, just to note we don't have any monitoring wrapper code around the task function so I don't believe it would report anything as the code currently is without that. To do that we'd need to add (I don't think we _need_ to add it but just wanted to clarify that):

```
import newrelic.agent


def identify_sql_injections():
    app = newrelic.agent.register_application(timeout=10)
    with newrelic.agent.BackgroundTask(app, "identify_sql_injections"):
        settings = get_config()
        logger.info("Running identify sql injections task.")
        sql_queries, http_requests = _get_data_from_pixie(settings)
        filtered_sql_queries = _filter_data(sql_queries)
        sql_injections = _identify_sql_injections(filtered_sql_queries)
        filtered_http_requests = _filter_http_data(http_requests)
        xss_events = _identify_xss(filtered_http_requests)
        _submit_nr_events(settings, sql_injections + xss_events)
    newrelic.agent.shutdown_agent(timeout=10)
```